src/app/modules/core/core.component.ts

```diff
@@ -14,9 +14,12 @@ export class CoreComponent implements OnInit {
     ngOnInit(): void {
         this.title = 'Inbox';
         this.access.verifyAccess().subscribe((data) => {
+            console.log(data);
```

Comment:
i had removed console line from my code. don't know why this was not updated in commit. may be i forget to save this file.